risc0/zkvm/src/serde/serializer.rs

```diff
@@ -18,6 +18,8 @@ use risc0_zkvm_platform::WORD_SIZE;
 
 use super::err::{Error, Result};
 
+use crate::alloc::string::ToString;
```

Comment:
TIL the guest does use `alloc` in the inverse of this PR: https://github.com/risc0/risc0/blob/8753f2c712280c2688e6b4492e8184e4572882fc/risc0/zkvm/src/serde/deserializer.rs#L15 
Should we ensure we can do a round trip of `NaiveDate` or perhaps some more generic type in tests?

Reply:
I did a round-trip test locally and this didn't require any code changes. Now that i'm looking at this more closely,  round-trip tests are in the mod.rs so I'll remove this test and add a round trip test there. Thanks!